packages/template-retail-react-app/app/pages/checkout/partials/shipping-address-selection.jsx

```diff
@@ -291,55 +291,75 @@ const ShippingAddressSelection = ({
                                     spacing={4}
                                     gridAutoFlow="row dense"
                                 >
-                                    {customer.addresses?.map((address, index) => (
-                                        <React.Fragment key={address.addressId}>
-                                            <RadioCard value={address.addressId}>
-                                                <ActionCard
-                                                    padding={0}
-                                                    border="none"
-                                                    onRemove={() =>
-                                                        removeSavedAddress(address.addressId)
-                                                    }
-                                                    onEdit={() => toggleAddressEdit(address)}
-                                                    editBtnRef={editBtnRefs[address.addressId]}
-                                                    data-testid={`sf-checkout-shipping-address-${index}`}
-                                                >
-                                                    <AddressDisplay address={address} />
-                                                </ActionCard>
-                                                {/*Arrow up icon pointing to the address that is being edited*/}
+                                    {customer.addresses?.map((address, index) => {
+                                        const editLabel = formatMessage(
+                                            {
+                                                defaultMessage: 'Edit {address}',
+                                                id: 'account_addresses.label.edit_button'
```

Comment:
I think let's not reuse this message id because it's intended for another page. How about duplicating it? Or refactor it to a more global location.

Reply:
using a more global id, `shipping_address.label.edit_button`